### ISSUE TITLE
Designer - when copying and pasting fields contained by layout, they should get new object names

### ISSUE THREAD

**Issue by 7ymekk** [fralx/LimeReport#284]
tl;dr; When copying and pasting e.g. Layouts with TextItems inside, they're being copied with same object names which leads to funny behavior in designer when operating on these fields. Or maybe the issue is not about object names but by layouts? I'm not sure in here

1. Create new page
2. put 3 TextItems all in 1 row. They'll be named TextItem1, TextItem2, TextItem3
3. select all of them with shift, then ctrl+C and ctrl+V. TextItem4, TextItem5 and TextItem6 appeared in Report structure. move them below to create 2nd row
4. select TextItem1, 2 and 3 using shift, right-click and pick 'Create Horizontal Layout'
5. select HorizontalLayout on Report Structure
6. press ctrl+C, click on empty page space (to unselect HorizontalLayout) and press ctrl+V
7. move HorizontalLayout, as you can see: there're 2 layouts now
8. notice that both layouts have fields with same names: Layout1[TextItem1, TextItem2 and TextItem3], Layout2[TextItem1, TextItem2 and TextItem3]
9. now try removing fields from one and another layout by clicking on them and pressing 'delete' on a keyboard. Sometimes when removing TextItem3 from Layout2, Designer removes TextItem3 from Layout1 and vice versa, when trying to remove some field from Layout1 it removes from Layout2. Sometimes it removes properly.
10. try copying Layout 10 times and remove fields or select all and remove them. sometimes it leads Designer to a deadlock, sometimes it crashes immediately.

**Comment by fralx** [comment]
Fixed

**Comment by 7ymekk** [comment]
works good now, thanks
